bindings/python/pymongoarrow/api.py

```diff
@@ -100,3 +101,54 @@ def aggregate_arrow_all(collection, pipeline, *, schema, **kwargs):
         process_bson_stream(batch, context)
 
     return context.finish()
+
+
+def _arrow_to_pandas_efficient(arrow_table):
```

Comment:
nit: let's drop the "_efficient" since hopefully all our code is as efficient as it needs to be and I don't see us adding a _arrow_to_pandas_inefficient :)

Reply:
lol